app/main.py

```diff
@@ -1,23 +1,37 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from django.db.models import QuerySet, Q, F
+
+from db.models import LoyaltyProgram, LoyaltyProgramParticipant
 
 
 def all_loyalty_program_names() -> QuerySet:
-    pass
+    return LoyaltyProgram.objects.values_list(
+        "name", "bonus_percentage")
```

Comment:
Your brackets are in different styles. Remake it in one style. like 
`values_list(`
`...`
`)`

Reply:
Thanks for your review! I've added all changes.